msteams-platform/toolkit/TeamsFx-CLI.md

```diff
@@ -1,23 +1,462 @@
+|:----------------  |:-------------|:-------------|
+| `--env` | No | Select an existing environment for the project. |
+| `--folder -f` | No | Select root folder of the project. Defaults to `./`. |
+| `--teams-manifest-file -t` | No | Select the manifest template file path, defaults to `${folder}/appPackage/manifest.json`. |
+| `--entra-app-manifest-file -a` | No | Select the Entra manifest template file path, defaults to `${folder}/aad.manifest.json`. |
+| `--all -a` | No | List all collaborators. |
+
+### Scenarios for `teamsapp collaborator`
+
+* Grant permission
+
+  Grant permission for another Microsoft 365 account to collaborate on the Teams app:
+
+  ```typescript
+  teamsapp collaborator grant -i false --teams-manifest-file ./appPackage/manifest.json --env dev --email other@email.com
+  ```
+
+  After you receive the required permission, project creator and collaborators can share the project with the new collaborator by GitHub, and the new collaborator can have all the permissions for Microsoft 365 account.
+
+* Show permission status
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view Microsoft 365 account permission for specific env:
+
+  ```typescript
+  teamsapp permission status --env dev
+  ```
+
+* List all collaborators
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view all collaborators for specific env:
+
+  ```typescript
+  teamsapp collaborator status --env dev --list-all-collaborators
+  ```
+
+## `teamsapp update`
+
+Update the Teams app manifest to Teams Developer Portal.
+
+### Parameters for `teamsapp update -h`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--env-file` |  | Select an .env file that defines the variables to replace in the manifest template file. |
+| `--folder -f` |  | Select root folder of the project. Defaults to `./` |
+| `--manifest-file` |  | Specifies the Teams app manifest file path. Default value: `./appPackage/manifest.json`. |
+| `--package-file` |  | Specifies the zipped Teams app package file path. |
+| `--output-package-file` |  | Specifies the output zipped Teams app package file path. Default value: `./appPackage/build/appPackage.${env}.zip`. |
+| `--output-manifest-file` |  | Specifies the final output Teams app manifest file path. Default value: `./appPackage/build/manifest.${env}.json`. |
+
+## `teamsapp upgrade`
+
+Upgrade the project to work with the latest version of Teams Toolkit.
+
+### Parameters for `teamsapp upgrade -h`
+
+|Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--force -f` | No | Force upgrade the project to work with the latest version of Teams Toolkit. Defaults value: `false`. |
+
+## `teamsapp install`
+
+Sideload a given application package across Microsoft 365.
+
+### Parameters for `teamsapp install -h`
+
```

Comment:
It should be parameters for `teamsapp install`, remove `-h`. Same applies for other commands.

Reply:
Updated.